kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -529,6 +543,19 @@ def kube_cronjob_next_schedule_time(self, metric, scraper_config):
             else:
                 self.service_check(check_basename, self.OK, tags=tags)
 
+    def kube_job_status_active(self, metric, scraper_config):
+        for sample in metric.samples:
+            tags = [] + scraper_config['custom_tags']
+            for label_name, label_value in iteritems(sample[self.SAMPLE_LABELS]):
+                if label_name == 'job' or label_name == 'job_name':
+                    trimmed_job = self._trim_job_tag(label_value)
+                    tags.append(self._format_tag(label_name, trimmed_job, scraper_config))
+                else:
+                    tags.append(self._format_tag(label_name, label_value, scraper_config))
```

Comment:
Unless I miss something, this is already duplicated 4 times, it'd be nice to refactor this tag building logic.

Reply:
Thanks! Will do!
Actually,I will abandon this PR for another one more appropriate.